pmoves/services/github-runner-ctl/github/client.py

```
@@ -0,0 +1,422 @@
+        name = runner.get("name")
+        busy = runner.get("busy", False)
+        status = "busy" if busy else "online"
+
+        # Store for tracking
+        self._runner_status[str(runner_id)] = {
+            "id": runner_id,
+            "name": name,
+            "status": status,
+            "source": source,
+            "os": runner.get("os"),
+            "labels": [l.get("name") for l in runner.get("labels", [])],
```

Ruff (0.14.10)

Ambiguous variable name: `l`

(E741)